### PR TITLE
editorial: This sentence was hard to parse for me (recovery)

### PR DESCRIPTION
**Pull request by mirjak**
So here is an alternative proposal. However I also wonder why this paragraph is a MUST requirement while the previous statement (avoiding multiple samples of the same packet) is only a SHOULD? I would see it rather the other way around or both MUSTs.

### CODE REVIEW

File: draft-ietf-quic-recovery.md [hunk 1]
```diff
@@ -272,9 +272,10 @@ SHOULD NOT be used to update RTT estimates if it does not newly acknowledge the
 largest acknowledged packet.
 
 An RTT sample MUST NOT be generated on receiving an ACK frame that does not
-newly acknowledge at least one ack-eliciting packet.  A peer does not send an
-ACK frame on receiving only non-ack-eliciting packets, so an ACK frame that is
-subsequently sent can include an arbitrarily large Ack Delay field.  Ignoring
+newly acknowledge at least one ack-eliciting packet.  A peer does usally not
```
Comment (dtikhonov) [inline draft-ietf-quic-recovery.md]: s/usally/usually/

Also, I'd place "usually" before does:

> A peer usually does not send [...]

Comment (MikeBishop) [reply draft-ietf-quic-recovery.md]: "does not usually" would also be fine.  But not in the middle.

File: draft-ietf-quic-recovery.md [hunk 2]
```diff
@@ -272,9 +272,10 @@ SHOULD NOT be used to update RTT estimates if it does not newly acknowledge the
...
+send an ACK frame when only non-ack-eliciting packets are received. Therefore
+an ACK frame that contains only acknowledge for non-ack-eliciting packets is
```
Comment (MikeBishop) [inline draft-ietf-quic-recovery.md]:
```suggestion
an ACK frame that only contains acknowledgements for non-ack-eliciting packets
```
May require a rewrap

Comment (mirjak) [reply draft-ietf-quic-recovery.md]: thanks. done!

File: draft-ietf-quic-recovery.md (hunk 1 above)
Comment (mirjak) [reply draft-ietf-quic-recovery.md]: done

File: draft-ietf-quic-recovery.md [hunk 3]
```diff
@@ -272,9 +272,10 @@ SHOULD NOT be used to update RTT estimates if it does not newly acknowledge the
 largest acknowledged packet.
 
 An RTT sample MUST NOT be generated on receiving an ACK frame that does not
-newly acknowledge at least one ack-eliciting packet.  A peer does not send an
-ACK frame on receiving only non-ack-eliciting packets, so an ACK frame that is
-subsequently sent can include an arbitrarily large Ack Delay field.  Ignoring
+newly acknowledge at least one ack-eliciting packet. A peer usually does not
+send an ACK frame when only non-ack-eliciting packets are received. Therefore
+an ACK frame that only contains acknowledgements for non-ack-eliciting packets
+can subsequently include an arbitrarily large Ack Delay value.  Ignoring
```
Comment (martinthomson) [inline draft-ietf-quic-recovery.md]:
```suggestion
could include an arbitrarily large Ack Delay value.  Ignoring
```

File: draft-ietf-quic-recovery.md [hunk 4]
```diff
@@ -272,9 +272,10 @@ SHOULD NOT be used to update RTT estimates if it does not newly acknowledge the
 largest acknowledged packet.
 
 An RTT sample MUST NOT be generated on receiving an ACK frame that does not
-newly acknowledge at least one ack-eliciting packet.  A peer does not send an
-ACK frame on receiving only non-ack-eliciting packets, so an ACK frame that is
-subsequently sent can include an arbitrarily large Ack Delay field.  Ignoring
+newly acknowledge at least one ack-eliciting packet. A peer usually does not
+send an ACK frame when only non-ack-eliciting packets are received. Therefore
+an ACK frame that only contains acknowledgements for non-ack-eliciting packets
```
Comment (janaiyengar) [inline draft-ietf-quic-recovery.md]:
```suggestion
an ACK frame that contains acknowledgements for only non-ack-eliciting packets
```

Comment (MikeBishop) [reply draft-ietf-quic-recovery.md]: Technically, it could still acknowledge other packets as well -- the key is which packets are *newly* acknowledged, isn't it?
